bin/push-locales

```diff
@@ -51,10 +51,20 @@ GIT_AUTHOR_EMAIL: $GIT_AUTHOR_EMAIL
 GIT_COMMITTER_NAME: $GIT_COMMITTER_NAME
 GIT_COMMITTER_EMAIL: $GIT_COMMITTER_EMAIL
 
-This script passes arguments directly to Git commands. We can pass --dry-mode to test this script
-Without actually committing or pushing. Make sure to only pass arguments supported on both commit and push..
+This script passes arguments directly to Git commands. We can pass --dry-mode to test this script.
+Without actually committing or pushing. Make sure to only pass arguments supported on both commit
+and push.
+
 ARGS: $@
 """
 
 git commit -am "$MESSAGE" "$@"
+
+if [[ "$@" =~ '--dry-run' ]]; then
+  info """
+    Skipping 'git push' because '--dry-run' is in ARGS and we do not have git credentials.
```

Comment:
Technically this is not checking that we have credentials available or not, so maybe reword in a less specific way:
```suggestion
    Skipping 'git push' because '--dry-run' is in ARGS so we should not have git credentials.
```